corehq/apps/sms/tests/test_backends.py

```diff
@@ -321,6 +332,7 @@ def test_outbound_sms(
         self._test_outbound_backend(self.icds_backend, 'icds test', icds_send)
         self._test_outbound_backend(self.vertext_backend, 'vertex_test', vertex_send)
         self._test_outbound_backend(self.start_enterprise_backend, 'start_ent_test', start_ent_send)
+        self._test_outbound_backend(self.ivory_coast_mtn_backend, 'ivory_coast_mtn_test', ivory_coast_mtn_send)
```

Comment:
What does `'ivory_coast_mtn_test'` correspond to?  Is that just a dummy string for debugging?

Reply:
it can be any unique string, the test uses it to find the message (what's being tested here is that the message flows through the system without actually sending it)